easybuild/easyconfigs/y/Yambo/Yambo-4.1.3-CrayIntel-2016.11.eb

```diff
@@ -0,0 +1,46 @@
+# contributed by Luca Marsella (CSCS)
+easyblock = 'MakeCp'
```

Comment:
Why use `MakeCp` if you are setting `with_configure = 'True'`?
Why not `ConfigureMake`?
What do you think?